db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
 RolesConversion.new.gsa18f_approvers
+RolesConversion.new.gateway_roles
```

Comment:
does this need to be in seeds, or can we create in selectively for the specs that require this role?